wiki/redirect.yaml

```diff
@@ -785,8 +795,9 @@
 "user_configuration":                         "osu!_Program_Files/User_Configuration_File"
 "user_config":                                "osu!_Program_Files/User_Configuration_File"
 
-"academy":                     "osu!academy"
-"announcements/osu!academy":   "osu!academy"
+"academy":                     "community/video_series/osu!academy"
```

Comment:
The casing should usually match the folder's, else web has to do an extra url redirection for that.

Reply:
fair point. fixed